end-to-end-test/remote/specs/core/oncoprint.spec.js

```diff
@@ -341,7 +341,14 @@ describe('oncoprint', function() {
             function doCustomCaseOrderTest() {
                 // now we're on results page
                 waitForOncoprint(ONCOPRINT_TIMEOUT);
-                // open sort menu
+
+                // make sure we are in sample mode
```

Comment:
this is probably going to fail - what you should do instead of using click to open the dropdown is use the spec method `setDropdownOpen`.

also small thing, but the comment is wrong on 345 - should be `make sure we are in patient mode`